app/src/main/java/xterminators/spellingbee/gui/GuiView.java

```diff
@@ -65,7 +65,7 @@ public void InitUI() {
         standardFont = new Font("Helvetica", Font.BOLD, 16);
         smallFont = new Font("Helvetica", Font.BOLD, 13);
         guessKeyListener = new GuessKeyListener();
-        guiFunctions = new GuiFunctions(this);
+        guiController = new GuiController(this);
```

Comment:
This feels a little strange/concerning to me, but you know the GUI codebase far better than me, so if it makes sense to do it like this, then that's good.

Reply:
What's concerning about it? The view has a reference to the controller so that it can call the controller's functions. I'll admit the commit looks super weird because GuiController had been the name of the view and is now the controller.
That line should maybe be in a constructor instead of in InitUI() though.